### PR TITLE
Created BasketService Integration tests

### PR DESCRIPTION
**Pull request by MateuszHocholowski**

### CODE REVIEW

**Comment by RadekKaniecki** [comment]
BaskerService? :D

File: src/test/java/com/orzechazo/eshop/services/BasketServiceImplIT.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        DB_DEFAULT_BASKET_COUNT = bootstrapBasket.getBaskets().size();
+    }
+
+    @Test
+    void getBasketByBasketId() {
+        //when
+        BasketDto returnedDto = basketService.getBasketDtoByBasketId(DB_BASKET_ID);
+        //then
+        assertEquals(DB_BASKET_ID,returnedDto.getBasketId());
+        assertEquals(new BigDecimal("120"),returnedDto.getTotalPrice());
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/services/BasketServiceImplIT.java:46]: Use constant BASKET_1_TOTAL_PRICE

**Comment by MateuszHocholowski** [comment]
Corrected

File: src/test/java/com/orzechazo/eshop/bootstrap/tests/BootstrapBasket.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    public BootstrapBasket(BasketRepository basketRepository) {
+        this.basketRepository = basketRepository;
+    }
+    public void loadData() {
+        log.debug("loading Baskets...");
+
+        Basket basket1 = new Basket();
+        basket1.setBasketId(DB_BASKET_ID);
+        basket1.setTotalPrice(new BigDecimal("120"));
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/bootstrap/tests/BootstrapBasket.java:26]: BASKET1_TOTAL_PRICE should be created here

**Review by RadekKaniecki — APPROVED** [review]

**Pull request merged**